account/applications/methods/postApplications.md

```diff
@@ -14,18 +14,23 @@ Bandwidth's Account API leverages Basic Authentication with your Dashboard API C
+| `MsgCallbackUrl`                      | Yes, for `Messaging-V2`    | Url to receive [message events](../../../messaging/callbacks/messageEvents.md)                                                                                                                                                     |
+| `CallInitiatedCallbackUrl`            | Yes, for `Voice-V2`        | Url to receive [voice events](../../../voice/bxml/callbacks/about.md)                                                                                                                                                              |
+| `CallInitiatedMethod`                 | No                         | HTTP method for events sent to the `CallInitiatedCallbackUrl`.<br> <code class="post">POST</code> or <code class="get">GET</code><br>Default is <code class="post">POST</code>                                                     |
+| `CallStatusCallbackUrl`               | No                         | Url to receive [voice events](../../../voice/bxml/callbacks/about.md) **NOT** related to Initiated. Such as: rejected or hung up.                                                                                                  |
+| `CallStatusMethod`                    | No                         | HTTP method for events sent to the `CallStatusCallbackUrl`.<br> <code class="post">POST</code> or <code class="get">GET</code><br>Default is <code class="post">POST</code>                                                        |
+| `CallbackCreds`                       | No, but highly recommended | Basic auth credentials to apply to your message & voice events                                                                                                                                                                     |
+| `CallbackCreds.UserId`                | No, but highly recommended | Basic auth `UserId`                                                                                                                                                                                                                |
+| `CallbackCreds.Password`              | No, but highly recommended | Basic auth `Password`                                                                                                                                                                                                              |
+| `CallInitiatedFallbackUrl`            | No, for `Voice-V2`         | Url to receive [voice events](../../../voice/bxml/callbacks/about.md) URL is used when messages fail to arrive at `CallInitiatedCallbackUrl`                                                                                       |
```

Comment:
Just repeating here in case it gets lost in the conversation below: spoke with @lpinheiro-daitan , and he agrees that for these new fields, just mark "Mandatory" column as "No". That maintains the precedent set by other fields that are optional for voice and meaningless for messaging, (like `CallStatusCallbackUrl`)